go.mod

```diff
@@ -1,46 +1,48 @@
 module k8s.io/node-problem-detector
 
-go 1.20
+go 1.21
+
+toolchain go1.21.4
```

Comment:
I think this can be removed.

Reply:
k8s.io/test-infra@v0.0.0-20231117021824-70d582860a11 requires go@1.21 Do you think if we should bump to go 1.21?

Reply:
That would probably be best.

Reply:
I'll keep the change because it is required..

Reply:
I am curious, why is it required?

Reply:
I'm not sure if it is added by mistake.
https://github.com/kubernetes/test-infra/blob/59f3faaf5f0494159e6c56c04882db53563e2707/go.mod#L213

Reply:
we need to bump to 1.21 for sure, just not have the toolchain directive (I think I was not very clear).

Reply:
Oh. I see.

Reply:
Thanks for the review! It has been removed.